templates/volontulo/user_login.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
```

Comment:
shouldn't this be wrapped inside base.html template ?
at the same time base.html template should be simpler so we don't included items that we don't need at user login screen for instance?